### PR TITLE
Initial project structure

### PR DESCRIPTION
**Pull request by moamenhady**
This is the my first pull request to the UI project.
I have added the basic Angular project structure generated using Angular CLI.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# See https://docs.github.com/get-started/getting-started-with-git/ignoring-files for more about ignoring files.
```
Comment (tmohamed) [inline .gitignore:3]: .vscode directory would be added to .gitignore file

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Editor configuration, see https://editorconfig.org
```
Comment (tmohamed) [inline .editorconfig:1]: this file to be adde to .gitignore as well

File: src/app/app.component.html [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+<!-- * * * * * * * * * * * The content below * * * * * * * * * * * -->
+<!-- * * * * * * * * * * is only a placeholder * * * * * * * * * * -->
+<!-- * * * * * * * * * * and can be replaced.  * * * * * * * * * * -->
+<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
+<!-- * * * * * * * * * Delete the template below * * * * * * * * * -->
+<!-- * * * * * * * to get started with your project! * * * * * * * -->
+<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
+
+<style>
+  :host {
```
Comment (tmohamed) [inline src/app/app.component.html]: All CSS classes here to be move to app.component.css file instead

**Review by tmohamed — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (moamenhady) [reply .gitignore:3]: It's already in the auto-generated template.

**Pull request merged**